index.bs

```diff
@@ -385,6 +414,113 @@ spec: workers; urlPrefix: https://www.w3.org/TR/workers/#
         {{VideoConfiguration/contentType}}, if possible. Otherwise, the UA should
         presume that the video stream cannot produce alpha channel information.
       </p>
+
+      <p>
+        <pre class='idl'>
```

Comment:
This deviates from the style elsewhere in the doc. We generally first show the usage of some custom type in idl, describe that idl, then later define the custom type in an external section  (e.g. MediaConfiguration section shows usage of MediaDecodingType, but MediaDecodingType is defined after that in its own section).

Reply:
I think it would be good to leave breif descriptions of the dict members here, but have them reference external sections that describe their enums. e.g. 

"If specified, the hdrMetadataType describes the capability to interpret HDR metadata of the specified type."

could stay here and "specified type" could be made to link to a section further down that describes the enum and its members.